use-case-guides/workday-request-time-off/README.md

```diff
@@ -3,129 +3,96 @@ design_pattern_id: 2
 
-### Domain Permissions Required
+> ℹ️ **You will be creating at least two ISUs and two ISSGs, one in a staging environment, and one in production.**
 
-| Workday Web Service | Domain Permissions Required | Description |
-| --- | --- | --- |
-| Worker Data Query (WQL) | Get_References, Get_Workers, Get_Eligible_Time_Off_Plans, Get_Time_Off_Balance, Get_Positions | Permissions needed to retrieve worker information, including contact details, position, and time off balances. |
-| Time Off Request (ReST) | Put_Time_Off_Request, Get_Workers, Get_Positions | Permissions needed to submit time off requests and retrieve worker and position information. |
 
+2. Install the Custom Report below in Workday**
 
-> ℹ️ **You will be creating two ISUs, one in a staging environment, and one in production.**
+    Use the custom report definitions below to install the columns and prompt defaults for your custom report. This sets up calculated fields for the time off request and time off balance which are required for the plugin to work.
 
+    [Worker Details Moveworks Columns.xlsx](Plugin%20Template%20Request%20Time-Off%20in%20Workday%20081c4d522bf64bbead3697288dd46047/Worker_Details_Moveworks_Columns.xlsx)
 
-### Step 1: Create a Connector in Creator Studio
+    [Worker Details Moveworks Prompt.xlsx](Plugin%20Template%20Request%20Time-Off%20in%20Workday%20081c4d522bf64bbead3697288dd46047/Worker_Details_Moveworks_Prompt.xlsx)
 
-[Follow the authentication guide to create a working Workday connector in Creator Studio](https://developer.moveworks.com/creator-studio/resources/authentication-guide?id=workday)
+3. Share your Workday connector credentials with your Moveworks developer team.
 
-### **Step 2: Test required APIs with Postman**
+## Domain Permissions Required
 
-**Prerequisite: Install Custom Report in Workday**
+| Workday Web Service | Domain Permissions Required | Description |
+| --- | --- | --- |
+| Worker Data Query (WQL) | Get_References, Get_Workers, Get_Eligible_Time_Off_Plans, Get_Time_Off_Balance, Get_Positions | Permissions needed to retrieve worker information, including contact details, position, and time off balances. |
+| Time Off Request (REST) | Put_Time_Off_Request, Get_Workers, Get_Positions | Permissions needed to submit time off requests and retrieve worker and position information. |
 
-Use the custom report definitions below to install the columns and prompt defaults for your custom report
+![Domain%20Permissions](Plugin%20Template%20Request%20Time-Off%20in%20Workday%20081c4d522bf64bbead3697288dd46047/Screenshot%202024-07-12%20at%2012.49.29 PM.png)
 
-[Worker Details Moveworks Columns.xlsx](Plugin%20Template%20Request%20Time-Off%20in%20Workday%20081c4d522bf64bbead3697288dd46047/Worker_Details_Moveworks_Columns.xlsx)
+> ✅ **Inform your Creator Studio developer that they can now use your Workday credentials to build their connector and plugin** 🚀
```

Comment:
Style guide rule: the double emoji in one paragraph is excessive here

Reply:
Why is this not a step, but rather a ">"

Reply:
included as part of step and removed double emoji